### PR TITLE
Intoto summary

### PR DESCRIPTION
**Pull request by ultrasaurus**
Content taken from here: https://docs.google.com/document/d/1dcDu8XWHlJ47FsIh3M1NwihmgfO3iqjH_Dn5tkevdqg/edit

I moved around the sections a bit for readability and thinking a little about how this might have looked over time, if we had started with this process.

Note: this is both the in-toto assessment and a template.  We're planning to follow the same process for first 5 assessment (see https://github.com/cncf/sig-security/issues/167) -- we're not going to get too caught up in sticking to the format if we find unexpected differences, but trying to avoid revisiting process until we have done the first set.

### CODE REVIEW

File: assessments/projects/in-toto/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+*   Verify in-toto's supply chain with in-toto_
+    _[in-toto/issue#278](https://github.com/in-toto/in-toto/issues/278)
+*   Improve introductory documentation to clearly communicate security scope
+    [docs/issue#15](https://github.com/in-toto/docs/issues/15)
+*   Additional integrations, examples and/or documented case studies (such as:
+    [in-toto/issue#284](https://github.com/in-toto/in-toto/issues/284),
+    [roadmap#3](https://github.com/in-toto/ITE/issues/3))
+*   Consider encoding best practices in default implementation (such as
+    issue#287)
```
Comment (lumjjb) [inline assessments/projects/in-toto/README.md]: nit: Link for issue

File: assessments/projects/in-toto/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+*   Proceed with [CII silver
+    badge](https://bestpractices.coreinfrastructure.org/en/projects/1523?criteria_level=1)
+    & [roadmap](https://github.com/in-toto/in-toto/blob/develop/ROADMAP.md). 
+    Just a few open items, listed below:
+    * Projects MUST monitor or periodically check their external dependencies
+    (including convenience copies) to detect known vulnerabilities, and fix
+    exploitable vulnerabilities or verify them as unexploitable. 
+    * The projectMUST implement secure design principles, where
```
Comment (lumjjb) [inline assessments/projects/in-toto/README.md]: nit: projectMUST

File: assessments/projects/in-toto/README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+### CNCF recommendations
+
+* Identify UX Researcher to engage 2-3 companies or projects to evaluate the
+  time and effort required to integrate in-toto and recommend improvements
+* SIG-Security collaboration to document common supply chain threats and
+  complementary solutions that would cover security of all inputs
+
+### Project recommendations
+
+*   Verify in-toto's supply chain with in-toto_
```
Comment (lumjjb) [inline assessments/projects/in-toto/README.md]: Hmm, looks like this got cut off somehow?

File: assessments/projects/in-toto/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+In-toto ensures the integrity of the supply chain itself, specifically:
+1. What steps are to be carried in the supply chain
+2. Who can carry out each step
+3. How the artifacts between each step interconnect with each other
+
+_Maturity_
+* 1 public company case study: [Datadog Agent full
+pipeline](https://www.datadoghq.com/blog/engineering/secure-publication-of-datadog-agent-integrations-with-tuf-and-in-toto/)
+* multiple integrations: [Debian apt
```
Comment (lumjjb) [inline assessments/projects/in-toto/README.md]: nit: Caps for consistency

File: assessments/projects/in-toto/README.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+and process. Non-prescriptive approach leaves opportunity for additional
+higher-level tooling.
+
+**Analysis**: focused approach to mitigating risks in supply chain which are
+underserved by other tech. Threats are well-understood and mitigated, including
+a reasonable degradation approach. Solid code processes; resiliency would
+benefit from additional participants.  
+
+All questions from reviewers were addressed in
+[self-assessment](self-assessment.md) with non-critical issues captured as
```
Comment (lumjjb) [inline assessments/projects/in-toto/README.md]: This link is currently broken, should we include this file in the same PR as well?

File: assessments/projects/in-toto/README.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+## Recommendations
+
+To address supply chain vulnerabilities, companies need more than technology,
+they will need to adopt or formalize effective policies and procedures. 
+
+### CNCF recommendations
+
+* Identify UX Researcher to engage 2-3 companies or projects to evaluate the
+  time and effort required to integrate in-toto and recommend improvements
+* SIG-Security collaboration to document common supply chain threats and
```
Comment (pragashj) [inline assessments/projects/in-toto/README.md]: Please elaborate or reference to the common threats.

File: assessments/projects/in-toto/README.md (hunk 3 above)
Comment (ultrasaurus) [reply assessments/projects/in-toto/README.md]: good catch -- that was italic where I copied from -- removed!

File: assessments/projects/in-toto/README.md (hunk 5 above)
Comment (ultrasaurus) [reply assessments/projects/in-toto/README.md]: LOL.  filename had typo!  file merged in prior PR -- I just renamed to match this link.

File: assessments/projects/in-toto/README.md (hunk 6 above)
Comment (ultrasaurus) [reply assessments/projects/in-toto/README.md]: @SantiagoTorres will write up an SIG-Security proposal issue for this

**Comment by justincormack** [comment]
Needs a rebase...

File: assessments/projects/in-toto/README.md (hunk 6 above)
Comment (ultrasaurus) [reply assessments/projects/in-toto/README.md]: added link to the issues @SantiagoTorres wrote up https://github.com/cncf/sig-security/issues/224

**Pull request merged**